src/Swarm/TUI/View.hs

```diff
@@ -239,7 +239,7 @@ drawMenu isPaused viewingBase mode
       [ ("c", "recenter") | not viewingBase ]
 
     keyCmdsFor (Just InfoPanel)  =
-      [ ("↓↑/Pg{Up,Dn}/Home/End", "navigate")
+      [ ("↓↑/Pg{Up,Dn}/Home/End/{j,k}", "navigate")
```

Comment:
Should we use the same help as the world panel, e.g. `↓↑ / jk` ?

Reply:
I also could not decide. I have no problems with a version using less characters. Will update it!